.github/workflows/release_sdk_workaround.yml

```diff
@@ -0,0 +1,100 @@
+name: Release SDK Library - Workaround
+
+# This is a copy of `release.py` with a workaround for an issue with x86_64 binaries.
+# https://github.com/bbqsrc/cargo-ndk/issues/94
+
+on:
+  workflow_dispatch:
+    inputs:
+      rust-checkout-ref:
+        description: 'The branch, tag or SHA to checkout on the rust sdk.'
+        required: true
+        default: 'main'
+      sdk-version:
+        description: 'The new version for the sdk library.'
+        required: true
+
+
+jobs:
+  build_native:
+    name: Build and generate sdk aar
+    runs-on: macos-latest
+
+    concurrency:
+      group: ${{ github.ref }}-${{ github.job }}
+      cancel-in-progress: true
+
+    steps:
+
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
+      - name: Install android sdk
+        uses: malinskiy/action-android/install-sdk@release/0.1.2
+
+      - name: Install android ndk
+        uses: nttld/setup-ndk@v1
+        id: install-ndk
+        with:
+          ndk-version: r22b
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@nightly
+        with:
+          toolchain: 1.67.0
```

Comment:
Maybe we could use 1.68 here, but I'm not sure and testing it is quite time consuming.